Applications/WebEditor.qmd

```diff
@@ -0,0 +1,98 @@
+    Under the *Processes* section, users can find a comprehensive list of openEO processes specifically designed for Earth Observation processing. These processes operate on individual values within an array, accepting and returning a single value.
+
+    The *UDF Runtimes* section provides information on the available environments or platforms where User Defined Functions (UDFs) can be executed. Currently, the python runtime is available during this stage of development.
+
+    In the *Export File Formats* section, users are guided on the supported output formats within openEO. Clicking on each format provides a detailed summary of its associated parameters.
+
+2. **Help**
+    
+    The *Help* icon at the top of the screen will provide you with a short tour of the main section of the editor. 
+
+3. **Wizard**
+
+    The *Wizard* is an experimental feature that will help you to create openEO processes in a simple way for some common use cases.
+
+4. **Server**
+
+    The Server icon will pop up a window giving the user detailed information on the server used for processing the created processes. 
+
+5. **Guest**
+
+    The *Guest* naming will be replaced with your username when logged in. The dropdown will provide with an option to Log in.
+
+6. **Features**
+
+    The basic functionalities that can be handy when creating the processes in openEO Web Editor is available in this row. These functionalities includes creating a new script, importing processes from external sources, exporting in another programming language, validating processes on the server side, editing process metadata, adding parameters, etc. 
+
+7. **Process Editor**
+   
+   This is the editor for the processes. We recommend to work in "**Visual Model**" mode, where you can create processing chains simply by adding collections and processes and connecting them with each other. The "**Code**" mode allows to see the generated JSON process, which is usually only needed if you want to run the process using another client library such as Python or R.
+
+   The area on it's right will later be used for previewing collections or inspecting the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.
+
+8.  **Log in**
+
+    As previously mentioned, it is necessary to log in to interact with the server. A new window will appear when attempting to log in, as demonstrated below. There are multiple methods available to authenticate with the server. Choose the option for which you possess valid credentials. Typically, the recommended choice is the Copernicus Data Space Ecosystem option. However, you also have the option to log in using EGI Check-in, where you can select your organisation account. For further information regarding various authentication methods or to seek assistance, you can always click on the "help" option at the top or contact us.
+
+    <centre><img src="_images/login.PNG"  width="80%" height="60%"></centre>
```

Comment:
Note: we'll have to redo this screenshot once EGI is removed as option

Reply:
Sure.

Reply:
updated